hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DropTable.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.debug;
+
+import org.apache.hadoop.hdds.cli.SubcommandWithParent;
+import org.rocksdb.ColumnFamilyDescriptor;
+import org.rocksdb.ColumnFamilyHandle;
+import org.rocksdb.RocksDB;
+import picocli.CommandLine;
+
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+/**
+ * Drop a column Family/Table in db.
+ */
+@CommandLine.Command(
+        name = "drop_column_family",
+        description = "drop column family in db."
+)
+public class DropTable implements Callable<Void>, SubcommandWithParent {
+
+  @CommandLine.Option(names = {"--column_family"},
+      description = "Table name")
+  private String tableName;
+
+  @CommandLine.ParentCommand
+  private RDBParser parent;
+
+  @Override
+  public Void call() throws Exception {
+    List<ColumnFamilyDescriptor> cfs =
+        RocksDBUtils.getColumnFamilyDescriptors(parent.getDbPath());
+    final List<ColumnFamilyHandle> columnFamilyHandleList =
+        new ArrayList<>();
+    try (RocksDB rocksDB = RocksDB.open(
+        parent.getDbPath(), cfs, columnFamilyHandleList)) {
+      byte[] nameBytes = tableName.getBytes(StandardCharsets.UTF_8);
+      ColumnFamilyHandle toBeDeletedCf = null;
+      for (ColumnFamilyHandle cf : columnFamilyHandleList) {
+        if (Arrays.equals(cf.getName(), nameBytes)) {
+          toBeDeletedCf = cf;
+          break;
+        }
+      }
+      rocksDB.dropColumnFamily(toBeDeletedCf);
```

Comment:
Just a question:
If tablename is not a columnfamily in DB, we pass null for dropColumnFamily, do we need to print some error message if table is not in DB?

And also for the success scenario, do we want to add some message?

Reply:
Done. Thanks for the suggestion.